### PR TITLE
Fix play all & shuffle not working on genres

### PR DESCRIPTION
**Pull request by viown**
**Changes**
Fix shuffle/play all not working on movies & TV show genres.

**Issues**
Fixes https://github.com/jellyfin/jellyfin-web/issues/5909

### CODE REVIEW

File: src/controllers/list.js [hunk 1]
```diff
@@ -851,6 +851,10 @@ class ItemsView {
             setTitle(null);
             getItem(params).then(function (item) {
                 setTitle(item);
+                if (item.Type == 'Genre') {
```
Comment (thornbill) [inline src/controllers/list.js:854]: Just for my own understanding, why is this change needed?

Comment (viown) [reply src/controllers/list.js:854]: Without the ParentId, media from other libraries matching the genre will be queued as well.

**Review by thornbill — APPROVED** [review]

**Comment by WhateverTheyCallMe** [comment]
This is still happening to me in browser and iOS apps

Server version
10.10.6
Web version
10.10.6 (c335a3024)
Build version
10.10.6

**Comment by viown** [comment]
@WhateverTheyCallMe Please file an issue with your reproduction steps as it works on my end.
